variables.tf

```diff
@@ -39,6 +40,22 @@ variable "services" {
   )
 }
 
+#
+# var.catalog_services is a Consul Terraform Sync provided additional input
+# variable for modules to execute on a catalog-services condition
+#
+# An example of the catalog_services input value:
+# catalog_services = {
+#   "api" = ["blue", "green"]
+#   "consul" = []
+#   "web" = ["tag"]
+# }
+#
+variable "catalog_services" {
```

Comment:
I think including `catalog_services` as a part of the base template for modules might be misleading to think it's a suggested route. This option seems like it would be more confusing the more condition variables we add to CTS. What do you think about separate branches or folders to exemplify the different conditions?

Reply:
Hmmm! I see what you're saying that it's not fitting for the non-default condition to be represented in a base template. My original intention with this change was to give an example of how a non-default condition should be documented so hadn't planned to add to this repo with more condition variables. But open to updating this with other conditions down the line.

I really like your idea of separate branches for different conditions! I was concerned with separate folders that users will have the extra step of deleting the other folders. Let me adjust the content of this PR and push up a separate branch containing the template for the catalog-services.

Reply:
:nod: yeah, I agree that deleting the extra folders adds another unnecessary step.

I see, sounds good about only showing one example of conditions and not expanding to each possible condition.